packages/core/helper-plugin/lib/src/hooks/useSelectionState/index.js

```diff
@@ -34,5 +34,5 @@ export const useSelectionState = (key, initialValue) => {
     }
   };
 
-  return [selections, { selectOne, selectAll, selectOnly }];
+  return [selections, { selectOne, selectAll, selectOnly, setSelections }];
```

Comment:
If this hook is only used in the ML what do you think of moving it into the upload package instead of the helper one?

Reply:
Can we keep it there for now? It could be used in any places where we can select element in a list (single / multiple etc...)

Reply:
Can you add a story then so we know we have this hook available?